spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe CommentsController, type: :controller do
+  let!(:post_0) { create(:post, message: "First post", id: 0) }
```

Comment:
How would you get the factory to handle that?
https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#sequences